common/config/azure-pipelines/integration-validation.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
           imageName: macos-latest
           nodeVersion: 20.x
         Linux_node_22_x:
-          imageName: macos-latest
+          imageName: ubuntu-22.04 # ubuntu-latest is 24.04, and breaks tests using sandbox
           nodeVersion: 22.x
         Windows_node_22_x:
           imageName: macos-latest
```

Comment:
fix this as well (windows pointing to mac)

Reply:
I fixed it.